templates/page-styleguide.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * Template Name: Style Guide
+ *
+ * @package ThemeScaffold
+ */
+
+get_header();
+?>
+
+<div class="uikit__container">
+
+	<h1 class="uikit__heading">
+		<div class="uikit__block">
+			<span><?php echo esc_html( get_the_title(), 'sep' ); ?></span>
```

Comment:
No clue what the second parameter does. It's not documented in the codex.

Reply:
gah, that's a holdover from a project. Nice catch!